sections/discussion/category-topic.tex

```diff
@@ -28,15 +27,16 @@ \subsection{Category-topic model}\label{sec:discussion_category_topic}
 	\label{tab:category_similarity}
 \end{table}
 
-Of observations to be made here, the second most similar pair, 'Friii' and 'Debat', is interesting to look at. \vejleder[inline]{fix sentence}
+The second most similar pair, 'Friii' and 'Debat', is interesting to look at, since 
+'Debat' has a specific writing style and 'Friii' does not.
```

Comment:
```suggestion
The second most similar pair, 'Friii' and 'Debat', is interesting to look at since 'Debat' has a specific writing style and 'Friii' does not.
```
Sætningen blev skåret over